webapp/src/Service/ImportExportService.php

```diff
@@ -148,9 +148,20 @@ public function importContestData($data, ?string &$message = null, string &$cid
 
         $starttime->setTimezone(new DateTimeZone(date_default_timezone_get()));
         $activateTime = new DateTime();
-        if ($activateTime > $starttime) {
+        if (array_key_exists('activate-time', $data)) {
+            $activatetimeValue = $data['activate-time'];
+            if (is_string($activatetimeValue)) {
```

Comment:
We have this logic somewhere else as well right? Maybe put it in Utils?

Reply:
I've rewrote the PR, so I'll resolve this.